vault/resource_jwt_auth_backend_role_test.go

```diff
@@ -477,6 +479,58 @@ func TestAccJWTAuthBackendRole_fullUpdate(t *testing.T) {
 						"verbose_oidc_logging", "false"),
 				),
 			},
+			// Repeat test case again to remove attributes like `bound_claims`
```

Comment:
Would you mind moving these test check funcs to a common set that can be shared between the various steps. Please use https://github.com/hashicorp/terraform-provider-vault/blob/e3b6f4602c74e7f3a3e9f1f96e4f031453c54792/vault/resource_pki_secret_backend_role_test.go#L72 as an example.

The resource name/path should also be factored out to a test variable called `resourceName`, e.g. https://github.com/hashicorp/terraform-provider-vault/blob/e3b6f4602c74e7f3a3e9f1f96e4f031453c54792/vault/resource_pki_secret_backend_role_test.go#L18

Thanks

Reply:
I reduced a bit of code reptition but I did not refactor out the test steps into a separate function yet. This seems like more of a refactoring that might make the size of this PR bigger.

Reply:
Ok, we will take care of that after merge. Thanks!